apps/rush-lib/src/cli/taskRunner/ProjectTask.ts

```diff
@@ -92,7 +90,11 @@ export default class ProjectTask implements ITaskDefinition {
 
       const currentDepsPath: string = path.join(this._rushProject.projectFolder, RushConstants.packageDepsFilename);
       if (fsx.existsSync(currentDepsPath)) {
-        lastPackageDeps = JsonFile.load(currentDepsPath) as IPackageDependencies;
+        try {
+          lastPackageDeps = JsonFile.load(currentDepsPath) as IPackageDependencies;
+        } catch (e) {
+          // Ignore - treat failing to load the file as the project being not built.
```

Comment:
>// Ignore - treat failing to load the file as the project being not built. [](start = 10, length = 74)

Should we issue a warning?  

Under what conditions is this expected to occur?

What guarantees that the problem will won't recur every single time the person does a build?

Reply:
I'll change it to issue a warning.